content/en/platform/corda/4.5/enterprise/app-upgrade-notes-enterprise.md

```diff
@@ -213,4 +213,4 @@ dependencies {
 ```
 
 
-* Workflow CorDapps.Please follow the instructions listed in step 5 of [Upgrading apps to Corda 4](app-upgrade-notes.md#cordapp-upgrade-finality-flow-ref).
+* Workflow CorDapps.Please follow the instructions listed in step 5 of [Upgrading apps to Corda 4](../../../../../../en/platform/corda/4.5/enterprise/app-upgrade-notes.html#cordapp-upgrade-finality-flow-ref).
```

Comment:
```suggestion
* Workflow CorDapps.Please follow the instructions listed in step 5 of [Upgrading apps to Corda 4](../../../../../../en/platform/corda/4.5/enterprise/app-upgrade-notes.html#step-5-security-upgrade-your-use-of-finalityflow).
```

Also this:
app-upgrade-notes-enterprise.html#recompiling-for-enterprise
needs to change to:
app-upgrade-notes-enterprise.html#re-compiling-for-corda-enterprise-45